app/controllers/application_controller.rb

```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 
-class ApplicationController < ActionController::API
+class ApplicationController < ActionController::Base
```

Comment:
Are you sure that it's a good idea to change it?
Looks like all our API endpoint are inherited from the `Base` controller

Reply:
Think, it is normal because `ActionController::API` does not give method `layout` for ActiveAdmin::Devise::SessionsController:Class